app/services/report/docx_action/experiment.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# rubocop:disable  Style/ClassAndModuleChildren
+module Report::DocxAction::Experiment
```

Comment:
Do we have any good practice about how to name services? I'd say it's probably better to use e.g. `app/services/reports/generate_docx/...` or something like that.

Reply:
In this approach we receive very complex names with duplicates naming. 
My structure expect, that we have main folder for reports. One main class for each type of report and folder with additional modules and classes. I will optimize it little bit.

Reply:
:+1: In any case, other folders use plural, so at least it should be `services/reports/...`